test/address.js

```diff
@@ -36,15 +36,6 @@ describe('address', function () {
       })
     })
 
-    fixtures.valid.forEach(function (f) {
-      it('parses (as chunks) ' + f.script.slice(0, 30) + '... (' + f.network + ')', function () {
-        var chunks = bscript.decompile(bscript.fromASM(f.script))
-        var address = baddress.fromOutputScript(chunks, networks[f.network])
-
-        assert.strictEqual(address, f.base58check)
-      })
-    })
```

Comment:
ACK,  this was weird anyway.

But it will need to be marked as breaking change if it has truly been removed (and not just something we're trying to "discourage").